lib/rules/web/admin_console/4.1/common_ui_elements.xyaml

```diff
@@ -37,7 +40,7 @@ click_button:
 click_button_text:
   element:
     selector:
-      xpath: //a[text()='<button_text>']
+      xpath: //button[text()='<button_text>']
```

Comment:
I'm not sure why `click_button_text` has definition of `//a[text()='<button_text>']`, changed to `//button`

Reply:
Not sure if other case is using this rule for 4.1

Reply:
I found these rules referencing `click_button_text`
```
$ grep -B 5 -nrE 'click_button_text' lib/rules/web/admin_console/4.1/
lib/rules/web/admin_console/4.1//route.xyaml-114-select_all_filters:
lib/rules/web/admin_console/4.1//route.xyaml-115-  params:
lib/rules/web/admin_console/4.1//route.xyaml-116-    button_text: Select All Filters
lib/rules/web/admin_console/4.1//route.xyaml:117:  action: click_button_text
--
lib/rules/web/admin_console/4.1//node.xyaml-67-set_affinity_operator:
lib/rules/web/admin_console/4.1//node.xyaml-68-  params:
lib/rules/web/admin_console/4.1//node.xyaml-69-    option: Operator
lib/rules/web/admin_console/4.1//node.xyaml-70-    button_text: <operator>
lib/rules/web/admin_console/4.1//node.xyaml-71-  action: click_node_affinity_modal_dropdown
lib/rules/web/admin_console/4.1//node.xyaml:72:  action: click_button_text
--
lib/rules/web/admin_console/4.1//node.xyaml-73-set_affinity_effect:
lib/rules/web/admin_console/4.1//node.xyaml-74-  params:
lib/rules/web/admin_console/4.1//node.xyaml-75-    option: Effect
lib/rules/web/admin_console/4.1//node.xyaml-76-    button_text: <effect>
lib/rules/web/admin_console/4.1//node.xyaml-77-  action: click_node_affinity_modal_dropdown
lib/rules/web/admin_console/4.1//node.xyaml:78:  action: click_button_text
--
lib/rules/web/admin_console/4.1//common_ui_elements.xyaml:40:click_button_text:
```

Reply:
I think only case OCP-23044 is using it, I tried to run regression but found this case is moved to Not Automated

Reply:
OK, we can update it in future when OCP-23044 moved back to automated.

Reply:
ok, also literally it should be button rather than a, let's update again if needed